crates/rpc/src/cairo/ext_py/ser.rs

```diff
@@ -339,124 +328,131 @@ impl TryFrom<BlockId> for BlockHashNumberOrLatest {
 
 #[cfg(test)]
 mod tests {
-    use crate::cairo::ext_py::ser::NoncesWrapper;
+    use super::*;
+    use crate::cairo::ext_py::ser::Nonces;
     use pathfinder_common::{
         felt, {ContractAddress, ContractNonce},
     };
-    use std::collections::HashMap;
-
-    #[test]
-    fn serialize_some_updates() {
-        use super::{ContractUpdatesWrapper, StorageDiff};
-        use pathfinder_common::{StorageAddress, StorageValue};
-        use std::collections::HashMap;
-
-        let expected = r#"{"0x7c38021eb1f890c5d572125302fe4a0d2f79d38b018d68a9fcd102145d4e451":[{"key":"0x5","value":"0x0"}]}"#;
-        let map = {
-            let mut map = HashMap::new();
-            map.insert(
+
+    mod storage_updates {
+        use super::*;
+
+        #[test]
+        fn none() {
+            let result = serde_json::to_string(&StorageUpdates(None)).unwrap();
+            assert_eq!(result, "{}");
+        }
+
+        #[test]
+        fn empty() {
+            let result = serde_json::to_string(&StorageUpdates(None)).unwrap();
+            assert_eq!(result, "{}");
+        }
```

Comment:
There was a comment here saying that empty and none should be encoded differently, however the implementation did not actually differentiate between them. I matched the code behaviour and deleted the comment..

Reply:
Great! Only `{}` is correct since we're calling `len()` on the pending updates in Python which doesn't work on `None`.

Reply:
Though -- aren't these tests exactly the same now? I'd keep only the `none` variant.